src/env.cc

```diff
@@ -76,6 +76,10 @@ void Environment::Start(int argc,
       reinterpret_cast<uv_handle_t*>(&idle_check_handle_),
       close_and_finish,
       nullptr);
+  RegisterHandleCleanup(
+      reinterpret_cast<uv_handle_t*>(&destroy_ids_timer_handle_),
+      close_and_finish,
+      nullptr);
```

Comment:
@trevnorris FYI, this comes from 1e44fd960fe38. I probably would have made the same mistake, but I wanted to let you know :)